openwillis/measures/video/eye_blink.py

```diff
@@ -0,0 +1,285 @@
+# author:    Georgios Efstathiadis
+# website:   http://www.bklynhlth.com
+
+# import the required packages
+import time
+import json
+import os
+import logging
+
+import cv2
+import numpy as np
+import pandas as pd
+from scipy.spatial import distance as dist
+from scipy.signal import find_peaks
+
+import mediapipe as mp
+
+logging.basicConfig(level=logging.INFO)
+logger=logging.getLogger()
+
+dir_name = os.path.dirname(os.path.abspath(__file__))
+config_path = os.path.abspath(os.path.join(dir_name, 'config/eye.json'))
+
+file = open(config_path)
+CONFIG = json.load(file)
```

Comment:
Load config file at local level, where we are consuming this information.

Loading it at global level, system ends up consuming memory for the entire duration, while for local memory being released once the function or block completes its execution.

Reply:
method like structure: Just have a separate method for this

def get_config():
    """
    ------------------------------------------------------------------------------------------------------

    This function loads and returns a dictionary containing the configuration settings for speech separation
    from a JSON file.

    Parameters:
    ...........
    None

    Returns:
    ...........
    measures : dict
        A dictionary containing the configuration settings for speech separation.

    ------------------------------------------------------------------------------------------------------
    """
    #Loading json config
    dir_name = os.path.dirname(os.path.abspath(__file__))
    measure_path = os.path.abspath(os.path.join(dir_name, 'config/speech.json'))

    file = open(measure_path)
    measures = json.load(file)
    return measures
    
    I will suggest to move it to commons as it will be used by almost all the functions. And pass 'config/speech.json' as an argument.
    
    But its ok for now, will refactor the code in future